src/modules/homepage.js

```diff
@@ -0,0 +1,103 @@
+      container.appendChild(noMoviesMessage);
+      return;
+    }
+
+    genreData.forEach((movie) => {
+      const movieCard = document.createElement('div');
+      const movieTitle = document.createElement('h2');
+      const movieImage = document.createElement('img');
+      const buttonsDiv = document.createElement('div');
+      const commentBtn = document.createElement('button');
+      const likeBtn = document.createElement('i');
+
+      movieTitle.textContent = movie.show.name;
+      if (movie.show.image) {
+        movieImage.src = movie.show.image.medium;
+      } else {
+        movieImage.src = bin;
+      }
+
+      commentBtn.textContent = 'Comments';
+      likeBtn.classList.add('fa', 'fa-heart');
+
+      movieCard.classList.add('card');
+      buttonsDiv.classList.add('commentLike');
+
+      movieCard.appendChild(movieTitle);
+      movieCard.appendChild(movieImage);
+      container.appendChild(movieCard);
+
+      buttonsDiv.appendChild(likeBtn);
+      buttonsDiv.appendChild(commentBtn);
+      movieCard.appendChild(buttonsDiv);
+    });
+  };
+
+  // Add event listeners to genre menu items
+  const allBtn = document.getElementById('All');
+  const dramaBtn = document.getElementById('drama');
+  const comedyBtn = document.getElementById('comedy');
+  const actionBtn = document.getElementById('action');
+  const romanceBtn = document.getElementById('romance');
+
+  allBtn.addEventListener('click', () => {
+    displayMoviesByGenre(AllData);
+    allBtn.style.backgroundColor = 'grey';
+    comedyBtn.style.backgroundColor = '';
+    actionBtn.style.backgroundColor = '';
+    romanceBtn.style.backgroundColor = '';
+    dramaBtn.style.backgroundColor = '';
+  });
+
+  dramaBtn.addEventListener('click', () => {
+    displayMoviesByGenre(dramaData);
+    allBtn.style.backgroundColor = '';
+    comedyBtn.style.backgroundColor = '';
+    actionBtn.style.backgroundColor = '';
+    romanceBtn.style.backgroundColor = '';
+    dramaBtn.style.backgroundColor = 'grey';
+  });
+
+  actionBtn.addEventListener('click', () => {
+    displayMoviesByGenre(actionData);
+    allBtn.style.backgroundColor = '';
+    comedyBtn.style.backgroundColor = '';
+    actionBtn.style.backgroundColor = 'grey';
+    romanceBtn.style.backgroundColor = '';
+    dramaBtn.style.backgroundColor = '';
+  });
+
+  comedyBtn.addEventListener('click', () => {
+    displayMoviesByGenre(comedyData);
+    allBtn.style.backgroundColor = '';
+    comedyBtn.style.backgroundColor = 'grey';
+    actionBtn.style.backgroundColor = '';
+    romanceBtn.style.backgroundColor = '';
+    dramaBtn.style.backgroundColor = '';
+  });
+
+  romanceBtn.addEventListener('click', () => {
+    displayMoviesByGenre(romanceData);
+    allBtn.style.backgroundColor = '';
+    comedyBtn.style.backgroundColor = '';
+    actionBtn.style.backgroundColor = '';
+    romanceBtn.style.backgroundColor = 'grey';
+    dramaBtn.style.backgroundColor = '';
+  });
```

Comment:
You could avoid repetitions by writing:
For example:
```
romanceBtn.classList.add('active');
```

Then you can add styling in style.css:
```
.active {
  color: ...
  background-color: ...
}
```